tests/workflows/test_tsc_ic_usd.py

```diff
@@ -0,0 +1,49 @@
+import os
+from argparse import Namespace
+
+import pandas as pd
+
+from mawpy.constants import (USER_ID, UNIX_START_T, UNIX_START_DATE, ORIG_LAT, ORIG_LONG, ORIG_UNC, STAY_LAT, STAY_LONG,
+                             STAY, STAY_DUR)
+from mawpy.workflows.tsc_ic_usd import tsc_ic_usd
+
+
+def test_tsc_ic_usd(tmp_path):
```

Comment:
it seems like all of these tests could be refactored to use a common function along the lines of:
```
test_workflow(input, expected_output, workflow_fun, params_kwargs) 
```
Doens't need to be in this PR, but that would cut down the amount of boilerplate code.